src/Compilers/CSharp/Portable/Binder/DecisionDagBuilder.cs

```diff
@@ -552,7 +554,7 @@ bool IsDerivedType(TypeSymbol possibleDerived, TypeSymbol possibleBase)
 
             return Tests.AndSequence.Create(tests);
 
-            bool tryMakeSubpatternMemberTests([NotNullWhen(true)] BoundPropertySubpatternMember? member, ref BoundDagTemp input)
+            bool tryMakeSubpatternMemberTests([NotNullWhen(true)] BoundPropertySubpatternMember? member, ref BoundDagTemp input, bool isLengthOrCount = false)
```

Comment:
nit: let's make the parameter non-optional, as there are only two callers of this local function